CodeEditModules/Package.swift

```diff
@@ -6,15 +6,30 @@ let package = Package(
     name: "CodeEditModules",
     defaultLocalization: "en",
     platforms: [
-        .macOS(.v10_15),
+        .macOS(.v11),
```

Comment:
Why the choice to bump the OS version from `10.15` to `11`?
It can be a great replacement for Xcode, since Xcode is (going) to require 11+.

```
Xcode 13 includes SDKs for iOS 15, iPadOS 15, tvOS 15, watchOS 8, and macOS Big Sur 11.3. The Xcode 13 release supports on-device debugging for iOS 9 and later, tvOS 9 and later, and watchOS 2 and later. Xcode 13 requires a Mac running macOS 11.3 or later.
```

Reply:
seems that the library needs v11. I think this can be a great starting point and I don't see us using this library in the future, so we are going to replace it anyway